dashboard/app/models/user.rb

```diff
@@ -422,7 +422,7 @@ def self.find_or_create_facilitator(params, invited_by_user)
 
   # a bit of trickery to sort most recently started/assigned/progressed scripts first and then completed
   has_many :user_scripts, -> {order "-completed_at asc, greatest(coalesce(started_at, 0), coalesce(assigned_at, 0), coalesce(last_progress_at, 0)) desc, user_scripts.id asc"}
-  has_many :scripts, -> {where hidden: false}, through: :user_scripts, source: :script
+  has_many :scripts, ->(script) {where(script.launched?)}, through: :user_scripts, source: :script
```

Comment:
I don't think I have this set up right. Not sure how to write this so its checking published_state of preview or stable